### PR TITLE
Modify notification scheduling based on session answer.

### PR DESCRIPTION
**Pull request by eschlange**
* Add new service and cache to store session answer data.
* Added specs for new service and cache.
* Fixed several existing specs to check for new logic.
* Fixed several formatting issues with existing specs.
* Added new file references to config files.
* Updated session controller to use new storage service.

[#111982123]

### CODE REVIEW

File: src/js/caches/LocalSessionCache.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+(function() {
+  'use strict';
+
+  function LocalSession(cache) {
+    this.KEY = 'localSession';
```
Comment (michaelwehrley) [inline src/js/caches/LocalSessionCache.js]: I don't think you need this...or do you?

File: src/js/controllers/SessionsController.js [hunk 2]
```diff
@@ -17,6 +17,8 @@
     this.currentSessionComplete = null;
     this.currentSessionNumber = null;
 
+    this.NUMBER_OF_CIGARETTES_QUESTION = '<h4>How many cigarettes do you smoke per day?</h4>';
```
Comment (michaelwehrley) [inline src/js/controllers/SessionsController.js]: picky, but do you need the html here or can it be in the view?

Comment (eschlange) [inline src/js/controllers/SessionsController.js]: Yes, the clinician wants the HTML tags in the questions that are saved.

File: src/js/caches/LocalSessionCache.js (hunk 1 above)
Comment (eschlange) [inline src/js/caches/LocalSessionCache.js]: Yes, it is required for the inherited resource cache features.

**Comment by michaelwehrley** [comment]
:all: :good: :guacamole:

**Comment by eschlange** [comment]
Thanks @michaelwehrley !

**Pull request merged**